test/fake/signup_client.go

```diff
@@ -0,0 +1,128 @@
+package fake
+
+import (
+	"encoding/json"
```

Comment:
can we organize these imports? 

i installed https://godoc.org/golang.org/x/tools/cmd/goimports - it works okay, does basic organization. also removes unused imports which is kind of nice...

Reply:
Done